src/vs/workbench/contrib/extensions/browser/extensionsActions.ts

```diff
@@ -60,6 +60,7 @@ import { IProductService } from 'vs/platform/product/common/productService';
 import { IFileDialogService, IDialogService } from 'vs/platform/dialogs/common/dialogs';
 import { IProgressService, ProgressLocation } from 'vs/platform/progress/common/progress';
 import { Codicon } from 'vs/base/common/codicons';
+import { EXTENSIONS_INSTALLVSIX_COMMAND_ID } from 'vs/workbench/contrib/extensions/browser/extensions.contribution';
```

Comment:
Like a wrapper around a command, we do this also in explorer, I think it makes sense.